src/__tests__/evaluator.test.ts

```diff
@@ -0,0 +1,80 @@
+import { Evaluator } from '../evaluator';
+import { Logger } from '../log';
+
+describe('Evaluator', () => {
+  it('should return identifier if bucketby attribute does not exist', async () => {
```

Comment:
Technically this does not return identifier if bucketBy attribute doesn't exist, buckets by the identifier if the bucketBy attribute doesn't exist and returns the appropriate variation value.